### PR TITLE
Plugins: Refactor tracing config

### PR DESCRIPTION
**Pull request by marefr**
Makes it possible to reuse tracing environment variables for plugins.

### CODE REVIEW

**Review by wbrowne — APPROVED** [review]
LGTM

File: pkg/services/pluginsintegration/pluginconfig/envvars.go [hunk 1]
```diff
@@ -144,19 +145,24 @@ func (p *EnvVarsProvider) secureSocksProxyEnvVars() []string {
...
-		fmt.Sprintf("GF_INSTANCE_OTLP_SAMPLER_PARAM=%.6f", p.cfg.Tracing.OpenTelemetry.SamplerParam),
-		p.envVar("GF_INSTANCE_OTLP_SAMPLER_REMOTE_URL", p.cfg.Tracing.OpenTelemetry.SamplerRemoteURL),
-	}
-	if plugin.Info.Version != "" {
-		vars = append(vars, fmt.Sprintf("GF_PLUGIN_VERSION=%s", plugin.Info.Version))
+		fmt.Sprintf("GF_INSTANCE_OTLP_ADDRESS=%s", tracingCfg.OpenTelemetry.Address),
+		fmt.Sprintf("GF_INSTANCE_OTLP_PROPAGATION=%s", tracingCfg.OpenTelemetry.Propagation),
+		fmt.Sprintf("GF_INSTANCE_OTLP_SAMPLER_TYPE=%s", tracingCfg.OpenTelemetry.Sampler),
+		fmt.Sprintf("GF_INSTANCE_OTLP_SAMPLER_PARAM=%.6f", tracingCfg.OpenTelemetry.SamplerParam),
+		fmt.Sprintf("GF_INSTANCE_OTLP_SAMPLER_REMOTE_URL=%s", tracingCfg.OpenTelemetry.SamplerRemoteURL),
```
Comment (hairyhenderson) [inline pkg/services/pluginsintegration/pluginconfig/envvars.go:161]: This is fine for now, but for what it's worth, I'd really love if we could instead use the standard OTel env vars here...

**Review by andresmgot — APPROVED** [review]
LGTM

File: pkg/services/pluginsintegration/pluginconfig/envvars.go (hunk 1 above)
Comment (marefr) [reply pkg/services/pluginsintegration/pluginconfig/envvars.go:161]: See discussion in https://github.com/grafana/grafana-plugin-sdk-go/pull/1014

**Pull request closed without merging**